rhizome/host/lib/vm_setup.rb

```diff
@@ -387,7 +387,17 @@ def cloudinit(unix_user, public_key, nics)
     FileUtils.chown @vm_name, @vm_name, vp.cloudinit_img
   end
 
-  def write_user_data(unix_user, public_key)
+  def generate_swap_config(swap_size_bytes)
+    return unless swap_size_bytes
```

Comment:
Can it do a bit of defense in depth here and make sure it's a normal integer? Alas, a metastasis of our prototype-era textual YAML splicing

Reply:
I changed it to `yq(swap_size_bytes.to_i)`. Is it enough confidence?

Reply:
I think there'd be a lot less going on if you did something like:

```
fail "BUG" unless swap_size_bytes.instance_of?(Integer)
```

thing is, `#to_i` is almost never what you want in Ruby because of how it tends to like making errors return `0`. And in this case, we probably don't have good cause to sometimes-parse input (is that true?) we can be a bit more strict about finding breakage in the model of the program (unless you want implied rounding and parsing, e.g. `Integer(3.14)` => `3`)

If we only anticipate integers, we can check the class and be strict.

Reply:
Thanks for the comment. Refactored it